zerver/tests/test_auth_backends.py

```diff
@@ -784,8 +784,11 @@ def social_auth_test(self, account_data_dict: Dict[str, str],
                 # it's definitely the GitHub authentication backend.
                 if self.AUTH_FINISH_URL == "/complete/github/":
                     self.assert_in_success_response(["Select account"], result)
-                    result = self.client_get(self.AUTH_FINISH_URL,
-                                             dict(state=csrf_state, email=account_data_dict['email']), **headers)
+                    user_data = dict(state=csrf_state,
+                                     email=account_data_dict['email'],
+                                     avatar_url=account_data_dict.get('avatar_url'))
```

Comment:
should be `account_data_dict.get('avatar_url', '')` I think? since we decided to use `''` to represent no avatar?

Reply:
hmm, there is a function, `get_account_data_dict`, that all the other tests are using to "build" this `account_data_dict`, and if no `user_avatar_url` is provided, then its value is `''`. But, I guess putting that here as well, makes sure its value is really this.